doc/contributing/release-process.rst

```diff
@@ -255,26 +259,33 @@ Leading up to the release
 
         https://gist.github.com/amercader/4ec55774b9a625e815bf
 
-     Other helpful commands are::
-
-        git branch -a --merged > merged-current.txt
-        git branch -a --merged ckan-1.8.1 > merged-previous.txt
-        diff merged-previous.txt merged-current.txt
+     But dread found changed the first step slightly to get it to work::
 
-        git log --no-merges release-v1.8.1..release-v2.0
-        git shortlog --no-merges release-v1.8.1..release-v2.0
+        git log --pretty=format:%s --reverse --no-merges release-v2.4.2...release-v2.5.0 -- | grep -Pzo "^\[#\K[0-9]+" | sort -u -n > issues_2.5.txt
 
 #. A week before the translations will be closed send a reminder email.
 
 #. Once the translations are closed, sync them from Transifex.
 
-   Pull the updated strings from Transifex, check them, compile and push as
-   described in the previous steps::
+   Pull the updated strings from Transifex
 
         tx pull --all --force
+
+   Check and compile them as before::
+
         paster check-po-files ckan/i18n/*/LC_MESSAGES/ckan.po
         python setup.py compile_catalog
-        git commit -am " Update translations from Transifex"
+
+    The compilation shows the translation percentage. Compare this with the new
+    languages directories added to ckan/i18n::
+
+        git status
+
+   ``git add`` any that are now over 5% translated and delete the rest (of the new ones).
```

Comment:
You can also do `tx pull --all --minimum-perc=5 --force` so you only get the relevant ones from Transifex, saving you checking the percentage yourself